packages/components/src/components/icon/brand/ic-brand-deriv-ez.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 32 32"><path fill="#FF444F" d="M0 6a6 6 0 0 1 6-6h20a6 6 0 0 1 6 6v20a6 6 0 0 1-6 6H6a6 6 0 0 1-6-6V6Z"/><path fill="#E12E3A" d="M6 0a6 6 0 0 0-6 6v.174l29.914-4.721A5.976 5.976 0 0 0 26 0H6ZM30.532 2.068 25.044 32H26a6 6 0 0 0 6-6V6c0-1.504-.553-2.88-1.468-3.932Z"/><path fill="#fff" d="M8.53 20.851v-9.702h6.552v1.834h-4.369v1.904h3.879v1.792h-3.879v2.338h4.69v1.834H8.53ZM23.316 12.661a21.5 21.5 0 0 0-1.064 1.204c-.41.485-.835 1.017-1.274 1.596a55.387 55.387 0 0 0-1.302 1.792c-.43.616-.812 1.204-1.148 1.764h4.942v1.834h-7.63v-1.302c.261-.476.579-1.003.952-1.582.383-.588.78-1.176 1.19-1.764a43.21 43.21 0 0 1 1.246-1.722c.43-.56.826-1.06 1.19-1.498h-4.354v-1.834h7.252v1.512Z"/></svg>
```

Comment:
Why are you adding DerivEZ icon?

Reply:
Deriv EZ icon change is because of the white line issue that we had previously.

Reply:
let's remove it then